test.js

```diff
@@ -1,18 +1,29 @@
 const test = require('ava');
 const { execFile } = require('child_process');
 
-const getDuplicates = require('./index');
+const { getDuplicates, getDuplicatesDetails } = require('./index');
 
 test('export test', (t) => {
   t.is(typeof getDuplicates, 'function', 'main export is a function');
 });
 
-test('basic API function test', (t) => {
+test('#getDuplicates: basic API function test', (t) => {
   const duplicates = getDuplicates(__dirname, 'supports-color');
   t.truthy(duplicates.length, 'duplicate detected');
   t.is(duplicates[0], 'supports-color', 'expected duplicate found');
 });
 
+test('#getDuplicatesDetails: basic API function test', async (t) => {
+  const duplicates = await getDuplicatesDetails(__dirname, 'supports-color');
+  const [{ name: nameOne }, { name: nameTwo }] = duplicates;
+  const [lowerVersion, higherVersion] = duplicates
+    .map(({ version }) => parseInt(version, 10))
```

Comment:
So I guess that the difference is already in the major version number, but where do these `supports-color` packages even come from?

Reply:
This is a seemingly ubiquitous package. And I guess that's why it is used here. I'm actually not sure, what I should think about that approach… But it works (for now). 😬